<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BISON_View</class>
 <widget class="QMainWindow" name="BISON_View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BISON View</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTabWidget::tab:disabled { width: 0; height: 0; margin: 0; padding: 0; border: none; }</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="3">
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item row="0" column="4">
        <widget class="QCommandLinkButton" name="commandLinkButton">
         <property name="styleSheet">
          <string notr="true">color: red;</string>
         </property>
         <property name="text">
          <string>No Link</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string>No Go</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSave_Telemetry"/>
    <addaction name="actionSave_Telemetry_2"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Images"/>
    <addaction name="actionSave_Images"/>
    <addaction name="actionExport_Telemetry"/>
   </widget>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>&amp;Connection</string>
    </property>
    <addaction name="actionSet_Server"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConnection"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_Telemetry">
   <property name="text">
    <string>Lo&amp;ad Telemetry</string>
   </property>
  </action>
  <action name="actionSave_Telemetry_2">
   <property name="text">
    <string>Sa&amp;ve Telemetry</string>
   </property>
  </action>
  <action name="actionLoad_Images">
   <property name="text">
    <string>L&amp;oad Images</string>
   </property>
  </action>
  <action name="actionSave_Images">
   <property name="text">
    <string>&amp;Save Images</string>
   </property>
  </action>
  <action name="actionExport_Telemetry">
   <property name="text">
    <string>E&amp;xport Telemetry</string>
   </property>
  </action>
  <action name="actionSet_Server">
   <property name="text">
    <string>S&amp;et Server</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
